test/pipelinerun_test.go

```diff
@@ -113,6 +114,32 @@ func TestPipelineRun(t *testing.T) {
 		// 1 from PipelineRun and 1 from Tasks defined in pipelinerun
 		expectedNumberOfEvents: 2,
 		pipelineRunFunc:        getHelloWorldPipelineRun,
+	}, {
+		name: "pipeline succeeds when task skipped due to failed condition",
```

Comment:
thanks for adding this! i think i derped a little bit when I suggested this be an end to end test vs. an example: i though the pipelinerun would end up failing (therefore wouldnt work as an example) but i forgot that a failing condition actually doesnt affect the success of the PipelineRun 🤦‍♀  anyway tho i think either is fine :D

Reply:
Well, this led me down a rabbit hole which led to #1130 which I think was what finally fixed the issue where creating and using a Condition immediately would always fail with a NotFound error. So, definitely worth it :angel: